npm/vite-dev-server/src/makeCypressPlugin.ts

```diff
@@ -16,14 +18,18 @@ export const makeCypressPlugin = (
 ): Plugin => {
   let base = '/'
 
+  const posixSupportFilePath = supportFilePath ? resolve(projectRoot, supportFilePath).replace(OSSepRE, '/') : undefined
+
+  const normalizedSupportFilePath = posixSupportFilePath ? `${base}@fs/${posixSupportFilePath}` : undefined
```

Comment:
Out of curiosity where does this `@fs` come from? I've seen it before for the node_modules but never really understood it.

Reply:
The proper functions to condition file calls in Vite are not yet exposed.
See `fileToUrl()` in `assets.ts`.
Maybe we can get Vite to expose some kind of util library?

Until then its the only way since we have our HTML in the wrong place (within node_modules)

Reply:
All good, just curious - good to know this comes from Vite.